_posts/AzerothCoreHostedTutorial.md

```diff
@@ -0,0 +1,144 @@
+- Configuring a Droplet on DigitalOcean from scratch
+- Building the Core
+- Loading the Database
+
+There is a Step-by-Step YouTube Video which walks you through this entire process. It's encouraged you follow along with that video, while using this tutorial as a reference guide.
+
+[![Tutorial Video](https://img.youtube.com/vi/lenMiDtbHhs/0.jpg)](https://www.youtube.com/watch?v=lenMiDtbHhs)
+
+## PC Apps You May Need:
+#### MySQL Clients:
+- [HeidiSQL](https://www.heidisql.com/)
+- [MySQL Workbench](https://www.mysql.com/products/workbench/)
+- [SQLYog](https://www.webyog.com/product/sqlyog) (Paid)
+
+#### SSH Clients:
+- DigitalOcean’s Console
+- [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/)
+- [Terminus](https://termius.com/) (Paid)
+
+#### FTP Clients:
+- [Filezilla](https://filezilla-project.org/)
+- [Terminus](https://termius.com/) (Paid)
+
+#### File Editors:
+- [Notepad++](https://notepad-plus-plus.org/)
+
+## Choosing our Droplet
+#### Setup new Droplet
+- 8GB RAM / 4 Cores
+- Debian 9
+- Choose Region
+- Enable Monitoring
+- One-time Password
+- Name Droplet | AzerothCore
+
+## Droplet Initial Configuration
+#### Change your Time Zone
+- Enter the command `timedatectl` to confirm the Droplet is set to UTC (00:00) time.
+- Before we can set the time zone, we first need to figure out what the code is for your time zone. Enter the command `timedatectl list-timezones` to see a list of all time zones. Hit Enter until you find your time zone, and make note of it. Hit *Control+C* once you're done looking for time zones.
+- Once you're back at the command line, enter `sudo timedatectl set-timezone YOUR_TIME_ZONE`
+- Verify that the Time Zone set correctly by entering `timedatectl`
+
+#### Add a new User
+- It's best practice to not use the *root* user for applications. Because of this, we're going create a user that will be responsible for all AzerothCore activities. Enter `adduser azcore` to create a new user.
+- Next, let's make sure this user has sudo permissions. Provision those permissions by using the following command `usermod -aG sudo azcore`
+
+#### Verify Sudo access for the new User
+- Let's log in as our new user `sudo su azcore`
+- Enter `sudo whoami` and then enter the password you chose for this user (may be different from your root password.
+- If it says *root*, then you're all set! Enter the command `exit` to return to your root user for the next steps.
+
+#### Installing Key Libraries & Additional Apps
+- Next we need to install a bunch of important libraries, applications, and tools needed to Azerothcore. Run the following command: `sudo apt-get update && sudo apt-get install git cmake make gcc g++ clang default-libmysqlclient-dev libssl1.0-dev libbz2-dev libreadline-dev libncurses-dev mysql-server libace-6.* libace-dev` Choose 'Y' for any prompts asking for additional space for the installation.
```

Comment:
since the command it's very long, maybe it makes sense to use triple apex ``` so it becomes:

```
sudo apt-get update && sudo apt-get install git cmake make gcc g++ clang default-libmysqlclient-dev libssl1.0-dev libbz2-dev libreadline-dev libncurses-dev mysql-server libace-6.* libace-dev
```

up to you